packages/framework/deprecated-agent-scheduler-container-runtime/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+    "extends": [
+        "@fluidframework/eslint-config-fluid/eslint7"
+    ],
+    "parserOptions": {
+        "project": ["./tsconfig.json", "./src/test/tsconfig.json"]
+    },
+    "rules": {
+        "@typescript-eslint/strict-boolean-expressions": "off"
```

Comment:
why disable this rule?

Reply:
Leftover from copy/paste, I'll clean it up